Cannot find module '[...]/node_modules/@yoichiro/vite-plugin-handlebars/dist/internal'

Hi,

I found this plugin via [vitejs/awesome-vite](https://github.com/vitejs/awesome-vite?tab=readme-ov-file#plugins) because frankly, all other handlebars plugins I've found and tried so far are quite lacking. 😞 

However, installing this plugin and then running `npm run dev` leads to the following error:

```
failed to load config from [...]/handlebars-test/vite.config.ts
error when starting dev server:
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '[...]/handlebars-test/node_modules/@yoichiro/vite-plugin-handlebars/dist/internal' imported from [...]/handlebars-test/node_modules/@yoichiro/vite-plugin-handlebars/dist/index.js
    at finalizeResolution (node:internal/modules/esm/resolve:260:11)
    at moduleResolve (node:internal/modules/esm/resolve:920:10)
    at defaultResolve (node:internal/modules/esm/resolve:1119:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:542:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:511:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:241:38)
    at ModuleJob._link (node:internal/modules/esm/module_job:126:49)
```

> ℹ️ I omitted the filepaths on my filesystem using the `[...]` notation.

Steps to reproduce:
1. Run `npm create vite@latest`
2. Choose `Vanilla` framework & language `TypeScript`
3. Run `npm install && npm install --save-dev @yoichiro/vite-plugin-handlebars`
4. Create a `vite.config.ts` with the following contents:

```typescript
import {defineConfig} from 'vite';
import handlebarsPlugin from '@yoichiro/vite-plugin-handlebars';

export default defineConfig({
  plugins: [
    handlebarsPlugin()
  ]
})
```

5. Run `npm run dev`

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@yoichiro/vite-plugin-handlebars",
-  "version": "1.1.0",
+  "version": "1.1.1",
   "author": {
     "name": "Yoichiro Tanaka",
     "email": "yoichiro6642@gmail.com",
@@ -36,7 +36,8 @@
     "dist"
   ],
   "exports": {
-    ".": "./dist/index.js"
+    ".": "./dist/index.js",
+    "./internal": "./dist/internal.js"
   },
   "type": "module",
   "keywords": [

